rrfs-test/validated_yamls/templates/obtype_config/msonet_winds_288.yaml

```diff
@@ -0,0 +1,476 @@
+         - filter: AcceptList  # accept back selected stations
+           apply at iterations: 0,1
+           where:
+           - variable: MetaData/mesonetProvider
+             is_in: ["VTDOT"]
+           - variable: MetaData/stationIdentification
+             is_in: ["VT002","VT001"]
+         - filter: AcceptList  # accept back selected stations
+           apply at iterations: 0,1
+           where:
+           - variable: MetaData/mesonetProvider
+             is_in: ["WIDOT"]
+           - variable: MetaData/stationIdentification
+             is_in: ["EKNW3","WAKW3","WI054","HAUW3","PAKW3","HSNW3","MNRW3"]
+         - filter: AcceptList  # accept back selected stations
+           apply at iterations: 0,1
+           where:
+           - variable: MetaData/mesonetProvider
+             is_in: ["WXforYou"]
+           - variable: MetaData/stationIdentification
+             is_in: ["H0269","H0313","H0300","H0080","H0346","H0164","H0005","H0177","H0041","H0498","H0338","H0062","H0244","H0203","H0237","H0030","H0322","H0560","H0140","H0447"]
+         - filter: AcceptList  # accept back selected stations
+           apply at iterations: 0,1
+           where:
+           - variable: MetaData/mesonetProvider
+             is_in: ["WYDOT"]
+           - variable: MetaData/stationIdentification
+             is_in: ["KBIT","KBRX","KHLD","KBFU","KCMS","KPER","KDRC"]
+
+         # Bounds Check
+         - filter: Bounds Check
+           apply at iterations: 0,1
+           filter variables:
+           - name: windEastward
+           - name: windNorthward
+           minvalue: -130
+           maxvalue: 130
+           action:
+             name: reject
+
+         # Gross Error Check
+         - filter: Background Check
+           apply at iterations: 0,1
+           filter variables:
+           - name: windEastward
+           function absolute threshold:
+           - name: ObsFunction/WindsSPDBCheck
+             options:
+               wndtype:    [ 288 ]
+               cgross:     [ 5.0 ]
+               error_min:  [ 1.0 ]
+               error_max:  [ 5.0 ]
+               variable: windEastward
+           action:
+             name: reject
+
+         # Gross Error Check
+         - filter: Background Check
+           apply at iterations: 0,1
+           filter variables:
+           - name: windNorthward
+           function absolute threshold:
+           - name: ObsFunction/WindsSPDBCheck
+             options:
+               wndtype:    [ 288 ]
+               cgross:     [ 5.0 ]
+               error_min:  [ 1.0 ]
+               error_max:  [ 5.0 ]
+               variable: windNorthward
+           action:
+             name: reject
+
+         # Time window filter
+         - filter: Domain Check
+           apply at iterations: 0,1
+           where:
+             - variable:
+                 name: MetaData/timeOffset
+               minvalue: -0.10
```

Comment:
Would it be better to use the exact time here instead of a time offset?

Reply:
IMO, no. But we can hear from anyone else or if you have a specific reason for doing it that way. My thinking is this is more generic and wouldn't require changing those exact dates every time you create a new yaml like when you are cycling DA. Keeping it like this would eliminate that extra step.

Reply:
Makes sense to me.